charts/datadog/values.yaml

```diff
@@ -510,6 +510,20 @@ clusterAgent:
     # clusterAgent.networkPolicy.create -- If true, create a NetworkPolicy for the cluster agent
     create: false
 
+    # clusterAgent.networkPolicy.flavor -- Flavor of the network policy to use.
+    # Can be:
+    # * kubernetes for networking.k8s.io/v1/NetworkPolicy
+    # * cilium     for cilium.io/v2/CiliumNetworkPolicy
+    flavor: kubernetes
+
+    cilium:
+      # clusterAgent.networkPolicy.cilium.dnsSelector -- Cilium selector of the DNS server entity
+      dnsSelector:
```

Comment:
I wonder if there's a way to avoid this duplication, maybe a default shared one if none of these are set?

Reply:
I feel that it would be smart to avoid the duplication of the whole `XXXagent.networkPolicy` sections.
I’m not sure there’s a use case for enabling `CiliumNetworkPolicy` on node agent, standard k8s `NetworkPolicy` on cluster checks runner and no network policy at all on the cluster agent.

Reply:
I have factorised the network policy settings in the common `datadog` section: https://github.com/DataDog/helm-charts/pull/99/commits/baae815b500a568d37c3042e9686af7506ce4e41